browser/net/brave_site_hacks_network_delegate_helper.cc

```diff
@@ -98,13 +100,6 @@ int OnBeforeStartTransaction_SiteHacksWork(net::URLRequest* request,
   if (IsBlockTwitterSiteHack(request, headers)) {
     return net::ERR_ABORTED;
   }
-  if (IsUAWhitelisted(request->url())) {
```

Comment:
this method doesn't appear to be used anywhere else so can be deleted if we decide we don't need this

Reply:
Deleted. Thanks!